.github/CHANGELOG.md

```diff
@@ -2,6 +2,27 @@
 
 <h3>New features since last release</h3>
 
+- Added the `DoubleExcitation` four-qubit operation, which is particularly useful for quantum
+  chemistry applications. [(#1121)](https://github.com/PennyLaneAI/pennylane/pull/1121)
```

Comment:
Minor: the second half of the sentence "which is particularly useful for quantum chemistry applications", sounds a bit weird to me in a changelog 🤔 

Suggest making it more specific (saying in 2-3 extra words why it is useful), or removing 'particularly'?